projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -186,10 +212,9 @@ export class AuthService implements OnDestroy {
     options?: PopupLoginOptions,
     config?: PopupConfigOptions
   ): Observable<void> {
-    return from(
-      this.auth0Client.loginWithPopup(options, config).then(() => {
-        this.refreshState$.next();
-      })
+    return from(this.auth0Client.loginWithPopup(options, config)).pipe(
+      mergeMap(() => this.getAccessTokenSilently(/* { cacheOnly: true } */)),
```

Comment:
The SPAJS SDK doesnt support this yet. To avoid unnecessary calls we need a way to get a token from the cache only, without triggering a request.